src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/send_location_request_test.cc

```diff
@@ -315,18 +315,33 @@ TEST_F(SendLocationRequestTest, Run_LocationImageValid_Success) {
   command_->Run();
 }
 
+TEST_F(SendLocationRequestTest, Run_LocationImageValid_Warnings) {
+  InitialSetup(message_);
+  (*message_)[strings::msg_params][strings::location_image] =
+      SmartObject(smart_objects::SmartType_Map);
+  (*message_)[strings::msg_params][strings::location_image][strings::value] =
+      "notavailable";
+  EXPECT_CALL(
+      mock_message_helper_,
+      VerifyImage(
+          (*message_)[strings::msg_params][strings::location_image], _, _))
+      .WillOnce(Return(mobile_apis::Result::WARNINGS));
+  FinishSetup();
+  command_->Run();
+}
+
 TEST_F(SendLocationRequestTest, Run_LocationImageInvalid_Cancelled) {
   InitialSetup(message_);
   (*message_)[strings::msg_params][strings::location_image] =
       SmartObject(smart_objects::SmartType_Map);
   (*message_)[strings::msg_params][strings::location_image][strings::value] =
-      "1";
+      " 1";
```

Comment:
Any reason?

Reply:
@jacobkeeler maybe it was done by mistake. Reverted by https://github.com/smartdevicelink/sdl_core/pull/2276/commits/51e40bf42f03ed368ac51123b77444313a2fe84d